### ISSUE TITLE
Codes for Loss Function

### ISSUE THREAD

**Issue by Bai426** [SHI-Labs/FcF-Inpainting#7]
Hi, this is a great job for inpainting. I am training my own dataset, but I didn't find the codes for the loss function described in the paper. Can you please tell where I can find it? Thank you very much！

**Comment by praeclarumjj3** [comment]
Hi @Bai426, thanks for your interest in our work! You can find the loss function implementation in the [`loss.py`](https://github.com/SHI-Labs/FcF-Inpainting/blob/main/training/losses/loss.py) file.

**Comment by Bai426** [comment]
Thank you very much!
